[OSDOCS#17131]: Release Notes CQA for 4.20 (#117190)
openshift/openshift-docs: modules/rn-ocp-release-fixed-issues.adoc

// Module included in the following assemblies:
// * release_notes/ocp-4-20-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="rn-ocp-release-fixed-issues_{context}"]
= Bug fixes

> [role="_abstract"]
> Review the list of issues resolved in this {product-title} release. You can see if issues affecting your clusters or environments are fixed.

Reviewer: ```suggestion
[role="_abstract"]
You can review the list of issues resolved for {product-title} {product-version} to see if any of the issues affect your clusters or environments are fixed.
```
